### PR TITLE
Review of /doc/automobile

### PR DESCRIPTION
**Pull request by remidhum**

### CODE REVIEW

File: automobile/car.md [hunk 1]
```diff
@@ -3,8 +3,8 @@
 The `Car` PROTO inherits from the [AckermannVehicle](ackermannvehicle.md) PROTO
 and extends it. It should be used with the [driver](driver-library.md) library
 in order to add a model of the engine, transmission, lights, gearbox and brake.
-The joint devices are automatically filled in with the appropriate devices in
-function of the transmission type set in the parameters.
+The joint devices are automatically filled in with the appropriate devices depending
+on the transmission type set in the parameters.
```
Comment (remidhum) [inline automobile/car.md]: Could also be `as a function of` .

File: automobile/lidar-sensors.md [hunk 2]
```diff
@@ -39,7 +39,7 @@ to speed up the simulation.
 The `Sick Lms 291` is a 1 layer lidar with a range of up to 80 meters and a
 field of view of up to 180 degrees.
 
-The model of the `IbeoLux` contains a spherical projection, a configurable fixed
+The model of the `Sick Lms 291` contains a spherical projection, a configurable fixed
```
Comment (remidhum) [inline automobile/lidar-sensors.md]: Not too sure about this one, it seemed odd to mention the previous LIDAR in the `Sick Lms 291` section ...

Comment (DavidMansolino) [inline automobile/lidar-sensors.md]: You are correct, this was indeed wrong.

**Comment by remidhum** [comment]
:+1:

**Comment by remidhum** [comment]
:+1:

**Comment by omichel** [comment]
:+1:

**Pull request merged**